semcore/wizard/src/Wizard.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import createComponent, { Component, Root, sstyled, styled } from '@semcore/core';
+import { Box } from '@semcore/flex-box';
+import { Text } from '@semcore/typography';
+import CheckM from '@semcore/icon/Check/m';
+import keyboardFocusEnhance from '@semcore/utils/lib/enhances/keyboardFocusEnhance';
+
+import style from './style/wizard.shadow.css';
+
+class WizardRoot extends Component {
+  static displayName = 'Wizard';
+  static style = style;
+  static defaultProps = {
+    currentStep: 1,
+  };
+
+  getStepProps() {
+    const { currentStep } = this.asProps;
+    return {
+      currentStep,
+    };
+  }
+
+  getStepperProps(props) {
+    const { currentStep, steps } = this.asProps;
+    const isActive = props.value === currentStep;
+    const isDisabled = steps.find((s) => s.value === props.value).disabled;
+
+    return {
+      currentStep,
+      steps,
+      'aria-disabled': isDisabled,
+      'aria-current': isActive,
+    };
+  }
+
+  render() {
+    const SWizard = this.Root;
+    const { Children, styles } = this.asProps;
+
+    return styled(styles)(
+      <SWizard render={Box}>
+        <Children />
+      </SWizard>,
+    );
+  }
+}
+
+function Sidebar(props) {
+  const { Children, styles, title } = props;
+  const SSidebar = Root;
+  const SSidebarHeader = Text;
+  return sstyled(styles)(
+    <SSidebar render={Box} {...props} role="menu">
+      <SSidebarHeader tag="div">{title}</SSidebarHeader>
```

Comment:
title - require? if `title` undefined?

Reply:
What about `<SSidebarHeader tag="div">{title}</SSidebarHeader>` -> `{title && <SSidebarHeader tag="div">{title}</SSidebarHeader>}`?

Reply:
Yes, I did, but I haven't push it yet)